### PR TITLE
Paperstorm comms

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: en-US/social/2017/paperstorm.lang [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+;Email not getting your MEP’s attention? Try dropping a flyer (or 10) on their town office. Help #fixcopyright with paperstorm.it
+Email not getting your MEP’s attention? Try dropping a flyer (or 10) on their town office. Help #fixcopyright with paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;One way to get EU lawmakers’ attention: drop thousands of digital leaflets. Help us #fixcopyright paperstorm.it
+One way to get EU lawmakers’ attention: drop thousands of digital leaflets. Help us #fixcopyright paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
```
Comment (flodolo) [inline en-US/social/2017/paperstorm.lang]: Add a note about the twitter nickname

File: en-US/social/2017/paperstorm.lang [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+;Yes! We dropped 10 000 000 leaflets on the EU Parliament! 🎉 We think [@comodinicachia] noticed :) Join paperstorm.it!
+Yes! We dropped 10 000 000 leaflets on the EU Parliament! 🎉 We think [@comodinicachia] noticed :) Join paperstorm.it!
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Props to [@name]! High score of [number] #paperstorm leaflets dropped! 💪 Are you a copyfighter? It’s your turn. paperstorm.it
+Props to [@name]! High score of [number] #paperstorm leaflets dropped! 💪 Are you a copyfighter? It’s your turn. paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
```
Comment (flodolo) [inline en-US/social/2017/paperstorm.lang]: Add a note about [number]

File: en-US/social/2017/paperstorm.lang [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+;CLICK FOR EUROPE. Join the #paperstorm & demand Europe’s politicians #fixcopyright. Drop virtual leaflets at paperstorm.it
+CLICK FOR EUROPE. Join the #paperstorm & demand Europe’s politicians #fixcopyright. Drop virtual leaflets at paperstorm.it
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;Clicks for europe. Clicks for copyright. Clicks for culture. Join the #paperstorm and demand #copyright reform in Europe 🇪🇺.
+Clicks for europe. Clicks for copyright. Clicks for culture. Join the #paperstorm and demand #copyright reform in Europe 🇪🇺.
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
```
Comment (flodolo) [inline en-US/social/2017/paperstorm.lang]: Is `[The Palace of Science and Culture]` a link or supposed to stay untranslated?

File: en-US/social/2017/paperstorm.lang [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+;Clicktivism unleashed. Copyfighters unite. Visit Paperstorm.it & tell EU parliament that citizens of the Internet demand reform.
+Clicktivism unleashed. Copyfighters unite. Visit Paperstorm.it & tell EU parliament that citizens of the Internet demand reform.
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
+;The fate of the Internet rests in your finger. Can you click our way to victory? Visit #Paperstorm & help demand copyright reform.
+The fate of the Internet rests in your finger. Can you click our way to victory? Visit #Paperstorm & help demand copyright reform.
+
+
+# Try to fit under 140 character, the string will be used on both Facebook and Twitter (if over 140 char, Facebook only)
```
Comment (flodolo) [inline en-US/social/2017/paperstorm.lang]: Grumpy old guy: should we add a note about <3?

Comment (TheoChevalier) [inline en-US/social/2017/paperstorm.lang]: Switching to ❤️ ;)

File: en-US/social/2017/paperstorm.lang (hunk 3 above)
Comment (TheoChevalier) [inline en-US/social/2017/paperstorm.lang]: Adding a note, it will be replaced with the FB page

**Pull request merged**